utils/src/main/java/org/jboss/windup/util/xml/XmlUtil.java

```diff
@@ -31,7 +31,8 @@
 
 public class XmlUtil
 {
-    private static Logger LOG = Logging.get(XmlUtil.class);
+	private static final XPathFactory XPATH_FACTORY = XPathFactory.newInstance();
```

Comment:
XPathFactory is not thread safe:
http://docs.oracle.com/javase/7/docs/api/javax/xml/xpath/XPathFactory.html

The code that uses this needs to be synchronized in order to use this pattern.